src/v/cluster/types.h

```diff
@@ -1362,6 +1364,16 @@ struct topic_table_delta {
         return model::topic_namespace_view(ntp);
     }
 
+    bool is_reconfiguration_operation() const {
```

Comment:
Are these two methods used?  I didn't find a usage site on a quick search of the PR diff.

Reply:
`controller_backend.cc:545`